bokehjs/test/integration/regressions.ts

```diff
@@ -4305,4 +4305,22 @@ describe("Bug", () => {
       await view.ready
     })
   })
+
+  describe("in issue #14280", () => {
+    it("triggers JS error when adding tile without defining range", async () => {
+      const osm = new TileRenderer({tile_source: osm_source.clone()})
+
+      const p0 = fig([300, 200], {
+        x_axis_type: "mercator",
```

Comment:
Although `figure` has implicit `DataRange1d` ranges, I would state that explicitly, because that's actually important for the test to be meaningful:
```suggestion
        x_range: new DataRange1d(),
        y_range: new DataRange1d(),
        x_axis_type: "mercator",
```